RoomMe/app/src/main/java/com/example/trocket/roomme/User.java

```diff
@@ -5,15 +5,26 @@
  */
 public class User {
 
-    public User() {
+    private String fullName;
+    private int age;
+    private double matchScore;
```

Comment:
I don't think this should be stored with the user because we won't always know who the match score is against. Instead, I do something like open an api endpoint that can generate a list of match scores given a user ID and a list of other user IDs